janusgraph-dist/src/assembly/static/bin/cassandra.in.sh

```diff
@@ -49,7 +49,7 @@ cassandra_storagedir="$CASSANDRA_HOME/data"
 # The java classpath (required)
 CLASSPATH="$CASSANDRA_CONF:$cassandra_bin"
 
-CLASSPATH="$CLASSPATH":"$CASSANDRA_HOME"/lib/slf4j-log4j12-1.7.5.jar
+CLASSPATH="$CLASSPATH":`ls "$CASSANDRA_HOME"/lib/slf4j-log4j12-*.jar`
```

Comment:
`$(...)` appears to be preferred to <code>\`...\`</code>: http://mywiki.wooledge.org/BashFAQ/082